cmd/security-secretstore-setup/README.md

```diff
@@ -39,3 +39,42 @@ make docker_security_secretstore_setup
 ```
 
 It should create a docker image with the name `edgexfoundry/docker_security_secretstore_setup:<version>-dev` if sucessfully built.
+
+## Debugging Tips
+
+* The _RevokeRootTokens_ in `cmd/security-secretstore-setup/res/configuration.toml` controls whether the root token used to populate Vault is deleted at when edgex-vault-worker is done. Set this to _false_ if you use edgex-vault-worker to populate the system and then want to debug either security-secretstore-setup or security-secrets-setup.
+
+* The edgex-vault-worker uses _composefiles_vault-config_ volume to store its token. To copy the root token from edgex-vault-worker, use
+
+    ```sh
+    docker run --rm -v composefiles_vault-config:/vault/config alpine:latest cat /vault/config/assets/resp-init.json > resp-init.json
+    ```
+
+* To verify the root token
+
+    ```sh
+    docker exec -ti edgex-vault sh -l
+    export VAULT_SKIP_VERIFY=true
+    export VAULT_TOKEN=s.xxxxxxxxxxxxxxxx
+    vault token lookup
+    ```
+
+    where `s.xxxxxxxxxxxxxxxx` is the _root_token_ member of `resp-init.json`
+
+* To explore the vault
+
+    ```sh
+    docker exec -ti edgex-vault sh -l
+    export VAULT_SKIP_VERIFY=true
+    export VAULT_TOKEN=s.xxxxxxxxxxxxxxxx
+    vault kv list secret/
+    ```
+
+    and drill down from there. To read a key use `vault read`.
```

Comment:
`vault kv get`

Reply:
```
edgex-vault:/# vault kv get /secret/edgex/redis/redis5
====== Data ======
Key         Value
---         -----
password    |sL58?wrNiTehO}.
username    redis5
edgex-vault:/# vault read /secret/edgex/redis/redis5
Key                 Value
---                 -----
refresh_interval    168h
password            |sL58?wrNiTehO}.
username            redis5
```

Will update to say `vault kv get` or `vault read`.